browser/ui/BUILD.gn

```diff
@@ -180,6 +180,15 @@ source_set("ui") {
       "views/web_discovery_dialog_view.h",
     ]
 
+    # Enable first_run dialog on Win.
+    # Upstream only includes it on mac/Linux.
```

Comment:
> Upstream only includes it on mac/Linux.

what's the reason? maybe it's buggy? maybe there's a chance to freeze the browser at startup or something like that?
can you dig into git blame and crbug discussion please to make sure nothing like that is going on here?

Reply:
Ok, I'll dig more and ask mailing list.

Reply:
FirstRun dialog provide two options.
One is for enabling metrics/crash reporting and the other is configuring for default browser.
On Windows, first option could be handled w/o FirstRun dialog by propagating from download site to installer.
Then, installer has related tag (`usagestats=1`).
And, default browser setting is handled during the welcome(chrome://welcome) steps.
I assume that chrome on Windows doesn't use FirstRun dialog because metrics/crash reporting can be set w/o it.

Reply:
That's a very good reasoning for not running it on Windows for Chrome. Thank you!